### ISSUE TITLE
tests: perl unit tests are failing on maintenance/13.0

### ISSUE THREAD

**Issue by JeGoi** [inverse-inc/packetfence#7965]
**Describe the problem**
```
unittest/UnifiedApi/Controller/Config/Certificates.t (Wstat: 256 Tests: 34 Failed: 1)
  Failed test:  27
  Non-zero exit status: 1
Files=208, Tests=9809, 1424 wallclock secs ( 1.83 usr  1.02 sys + 1320.56 cusr 147.30 csys = 1470.71 CPU)
Result: FAIL
make: Leaving directory '/usr/local/pf' SystemoutJSON:<nil> Systemerr:[Wed Jan 10 01:09:01 2024] Boolean.pm: Subroutine JSON::PP::Boolean::(++ redefined at /usr/share/perl5/overload.pm line 48.
[Wed Jan 10 01:09:01 2024] Boolean.pm: Subroutine JSON::PP::Boolean::(0+ redefined at /usr/share/perl5/overload.pm line 48.
[Wed Jan 10 01:09:01 2024] Boolean.pm: Subroutine JSON::PP::Boolean::(-- redefined at /usr/share/perl5/overload.pm line 48.
eth1.1: error fetching interface information: Device not found
eth1.2: error fetching interface information: Device not found
CN = radius
error 20 at 0 depth lookup: unable to get local issuer certificate
C = FR, ST = Radius, O = Example Inc., CN = Example Server Certificate, emailAddress = admin@example.org
error 7 at 0 depth lookup: certificate signature failure
140362504591168:error:0407008A:rsa routines:RSA_padding_check_PKCS1_type_1:invalid padding:crypto/rsa/rsa_pk1.c:70:
140362504591168:error:04067072:rsa routines:rsa_ossl_public_decrypt:padding check failed:crypto/rsa/rsa_ossl.c:769:
140362504591168:error:0D0C5006:asn1 encoding routines:ASN1_item_verify:EVP lib:crypto/asn1/a_verify.c:179:
C = FR, ST = Radius, O = Example Inc., CN = Example Server Certificate, emailAddress = admin@example.org
error 7 at 0 depth lookup: certificate signature failure
139810384439104:error:0407008A:rsa routines:RSA_padding_check_PKCS1_type_1:invalid padding:crypto/rsa/rsa_pk1.c:70:
139810384439104:error:04067072:rsa routines:rsa_ossl_public_decrypt:padding check failed:crypto/rsa/rsa_ossl.c:769:
139810384439104:error:0D0C5006:asn1 encoding routines:ASN1_item_verify:EVP lib:crypto/asn1/a_verify.c:179:
C = FR, ST = Radius, L = Somewhere, O = Example Inc., emailAddress = admin@example.org, CN = Example Certificate Authority
error 10 at 1 depth lookup: certificate has expired
C = FR, ST = Radius, O = Example Inc., CN = Example Server Certificate, emailAddress = admin@example.org
error 10 at 0 depth lookup: certificate has expired

#   Failed test '200 OK'
#   at unittest/UnifiedApi/Controller/Config/Certificates.t line 351.
#          got: '422'
#     expected: '200'
# Looks like you failed 1 test of 34.
```
Certificate issue  like for devel.

CRITICAL => It is blocking maintenance patch released.

**Mandatory informations**

 * Branch: maintenance/13.0
 * Scenario: unit test
 * Pipelines URL:
   * https://gitlab.com/inverse-inc/packetfence/-/jobs/5896777383
 
**Symptoms**
How the problem can be identified if it appears again ?

**Comment by jrouzierinverse** [comment]
Fixed by 7d767e12f64245014335affdd246470652b1c302

**Comment by JeGoi** [comment]
Hum, it looks like there is still an issue:
```
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests

Test Summary Report
-------------------
unittest/UnifiedApi/GenerateSpec.t                   (Wstat: 256 Tests: 4 Failed: 1)
  Failed test:  3
  Non-zero exit status: 1
serialized_unittests/db/apply_upgrade.t              (Wstat: 256 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
Files=208, Tests=9865, 1145 wallclock secs ( 1.69 usr  1.25 sys + 1306.55 cusr 160.32 csys = 1469.81 CPU)
Result: FAIL
make: Leaving directory '/usr/local/pf' SystemoutJSON:<nil> Systemerr:[Thu Jan 11 06:51:27 2024] Boolean.pm: Subroutine JSON::PP::Boolean::(0+ redefined at /usr/share/perl5/overload.pm line 48.
[Thu Jan 11 06:51:27 2024] Boolean.pm: Subroutine JSON::PP::Boolean::(++ redefined at /usr/share/perl5/overload.pm line 48.
[Thu Jan 11 06:51:27 2024] Boolean.pm: Subroutine JSON::PP::Boolean::(-- redefined at /usr/share/perl5/overload.pm line 48.
CN = radius
error 20 at 0 depth lookup: unable to get local issuer certificate
eth1.1: error fetching interface information: Device not found
eth1.2: error fetching interface information: Device not found

#   Failed test 'Testing SubType'
#   at unittest/UnifiedApi/GenerateSpec.t line 637.
# Compared $data->{"oneOf"}[2]{"\$ref"}
#    got : '#/components/schemas/PfdetectSubTypeRegex'
# expect : '#/components/schemas/PfdetectSubTypeSecurityOnion'
# Looks like you failed 1 test of 4.
C = FR, ST = Radius, O = Example Inc., CN = Example Server Certificate, emailAddress = admin@example.org
error 20 at 0 depth lookup: unable to get local issuer certificate
C = FR, ST = Radius, O = Example Inc., CN = Example Server Certificate, emailAddress = admin@example.org
error 20 at 0 depth lookup: unable to get local issuer certificate
```
